geozero/tests/postgis.rs

```diff
@@ -1,16 +1,34 @@
+mod pg {
+    fn get_db_string() -> String {
+        std::env::var("DATABASE_URL").unwrap()
+    }
+
+    #[cfg(feature = "with-postgis-postgres")]
+    pub fn get_conn() -> postgres::Client {
+        postgres::Client::connect(&get_db_string(), postgres::NoTls).unwrap()
```

Comment:
I would prefer using
`postgres::Client::connect(&get_db_string(), postgres::NoTls).unwrap()`
instead of `get_conn()` in the examples.
The goal of the tests are also being usage examples which should be usable with minimal adaption.

The `get_pool()` function is ok, since it makes the tests quite a lot shorter.

Reply:
fixed